draft-ietf-rats-corim.md

```diff
@@ -1429,9 +1429,173 @@ is extended using Endorsements etc. from the accepted tags.
+against the `measurement-map` from the Accepted Claims Set. If any entry
+does not match then the Reference Value does not match.
+
+The algorithm used to match the `measurement-values-map` entries
+is described below. It depends on whether the reference value is tagged with a
+CBOR tag {{-cbor}},
+and on the `measurement-values-map` key which identifies the entry.
+
+If the Reference Value `measurement-values-map` value starts with a CBOR tag
+then the Verifier MUST use the algorithm associated with that tag to match
+the entries.
+
+This specification defines the matching algorithm for some CBOR tagged reference
+values, which is described in sub-sections below.
+
+A CoRIM profile may define additional tags and their matching algorithms.
+
+If the Verifier does not recognize the Reference Value CBOR tag value then
+the Reference Value does not match.
+
+If the Reference Value is not tagged and the measurement-value-map key is a
+value with handling described in the sub-sections below,
+then the algorithm appropriate to that key is used to match the entries.
+
+If the Reference Value is not tagged, and the `measurement-values-map` key
+is not a value described below, then the entries are compared
+using binary comparison of their CBOR encoded values. If the values
+are not binary identical then the Reference Value does not match.
+
+Note that while specifications may extend the matching semantics using CBOR tags,
+there is no way to extend the matching semantics of key values.
+Any new keys requiring non-default comparison must add a CBOR tag to the
+reference value describing the desired behaviour.
+
+If the Reference Value contains an `authorized-by` field then the Verifier
+SHALL check for an `authorized-by` field in the Accepted Claims Set entry
+compared in the steps above. If the Accepted Claims Set key is not one of
+the keys from the reference value `authorized-by` field then the
+reference value does not match.
+
+If all checks above have been performed successfully then the Reference Value
+matches.
+
+##### Comparison for svn entries
+
+The value stored under `measurement-values-map` key 1 is an SVN, which must
+have type UINT.
+
+If the Reference value for `measurement-values-map` key 1 is an untagged UINT or
+a UINT tagged with #6.552 then an equality comparison is performed. If the value
+of the SVN in Accepted Claims Set is not equal to the value in the Reference
+Value then the Reference Value does not match.
+
+If the Reference value for `measurement-values-map` key 1 is a UINT tagged with
+#6.553 then a minimum comparison is performed. If the value of the SVN in
+Accepted Claims Set less than the value in the Reference Value then the
+Reference Value does not match.
+
+##### Comparison for digests entries
+
+The value stored under `measurement-values-map` key 2, or a value tagged with
+#6.TBD is a digest entry. It contains one or more digests, each measuring the
```

Comment:
```suggestion
#6.TBD is a digest entry.
It contains one or more digests, each measuring the
```

Reply:
I don't remember a decision to define a #6.TBD for `digests` value. Maybe worry about documenting it when we cross that bridge?